gef-ui/server/services/__tests__/api.test.js

```diff
@@ -8,6 +8,8 @@ afterEach(() => {
   jest.clearAllMocks();
 });
 
+const fakeMongoId = 'ABCDEF123456789000000000';
```

Comment:
Could we add in some tests to check false is returned from all these newly validated endpoints with the following mongo IDs: '../../../etc/passwd/', 'localhost', '{}', '[]', ''.

Might be worth just making a list of these as inputs and doing a test.each loop for each of the endpoints, something like:

```
const invalidFacilityUrnTestCases = [["123"], ["127.0.0.1"], ["{}"], ["[]"]];

describe("when application ID is invalid", () => {
  test.each(invalidMongoId)(
    "returns a 400 if you provide an invalid application ID: %s",
    async (facilityUrn) => {

      const response = await api.getApplication(invalidMongoId);

      expect(response).toEqual(false);
    }
  );
});
```